src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -311,7 +311,18 @@ function _parseModalAttributes(node) {
   _renameAttribute(node, 'ok-text', 'ok-title');
   _renameAttribute(node, 'center', 'centered');
 
-  node.attribs['ok-only'] = ''; // only show OK button
+  const hasOkTitle = _.has(node.attribs, 'ok-title');
+  const hasFooter = node.children.some(child =>
+    _.has(child.attribs, 'slot') && child.attribs.slot === 'modal-footer');
+
+  if (!hasFooter && !hasOkTitle) {
+    // markbind doesn't show the footer by default
+    node.attribs['hide-footer'] = '';
```

Comment:
does setting it to `true` erase the `=""` portion in `hide-footer=""`? might be nice if so, same for `ok-only`

Reply:
Nope, i just get this

![image](https://user-images.githubusercontent.com/3646725/77230161-3f23cd00-6bcd-11ea-87ed-8ca03c434423.png)